### PR TITLE
Add  “obsess” and “pnpgraph_present”

### PR DESCRIPTION
**Pull request by sidhartha-sankar**
https://jira.op5.com/browse/MON-10930
Add parameters like “obsess” and “pnpgraph_present” for the host and service

### CODE REVIEW

File: lmd/objects.go [hunk 1]
```diff
@@ -616,11 +616,13 @@ func NewHostsTable() (t *Table) {
 	t.AddColumn("notification_interval", StaticUpdate, IntCol, "Interval of periodic notification or 0 if its off")
 	t.AddColumn("notification_period", StaticUpdate, StringCol, "Time period in which problems of this host will be notified. If empty then notification will be always")
 	t.AddColumn("notifications_enabled", DynamicUpdate, IntCol, "Whether notifications of the host are enabled (0/1)")
+	t.AddColumn("obsess", DynamicUpdate, IntCol, "The obsessing over host")
```
Comment (sni) [inline lmd/objects.go]: better add new Columns at the end (right before the AddOptColumn...). Otherwise we have to update the test data as well.

Comment (sidhartha-sankar) [reply lmd/objects.go]: Updated. Thanks. But still benchmark_test.go fails.
--- FAIL: BenchmarkParseResultJSON
	benchmark_test.go:37: wrong result size, expected 181, got 185
--- FAIL: BenchmarkParseResultWrappedJSON
	benchmark_test.go:75: wrong result size, expected 181, got 185
Do I have to update this test case to 185?

Comment (sni) [reply lmd/objects.go]: exactly. btw, you can run the travis locally by `make citest`

Comment (sidhartha-sankar) [reply lmd/objects.go]: Yah sure. Thanks.

**Comment by sni** [comment]
Thinking about it, `obsess` is a naemon specific column and will lead to errors with other clients so it should be made optional.

**Comment by sorantis** [comment]
@sidhartha-sankar see this https://github.com/golang/lint/issues/397

**Comment by sidhartha-sankar** [comment]
Added changes to support for Naemon livestatus.
With very few changes in peer.go , we can support adding new livestatus vendors in future. 
In this way I think we may not restrict vendors to add name to livestatus versions, rather they can easily add name in lmd.ini file.

**Comment by sni** [comment]
That doesn't sound very reliable and wouldn't work automatically. Most setups i know name their sites after the location or the purpose, not literal "naemon". Also most other vendors have already a product suffix. So i'd say this is definitly the better approach.

**Comment by sidhartha-sankar** [comment]
Ok then. I reverted the changes and created a placeholder for naemon. We leave the PR like this until naemon livestatus name is changed. Later we can edit the name here.

**Comment by sidhartha-sankar** [comment]
The build is failing due to go version check error. I think rebuild might make it pass.

**Comment by sni** [comment]
merged, thanks.

**Pull request merged**